yt_dlp/extractor/axs.py

```diff
@@ -0,0 +1,83 @@
+from .common import InfoExtractor
+from ..utils import (
+    float_or_none,
+    js_to_json,
+    parse_iso8601,
+)
+
+
+class AxsIE(InfoExtractor):
+    IE_NAME = 'axs.tv'
+    _VALID_URL = r'https?://(?:www\.)?axs\.tv/(?:channel/[^/]+/)?video/(?P<id>[^?/]+)'
```

Comment:
exclude both queries and fragments with both character classes
```suggestion
    _VALID_URL = r'https?://(?:www\.)?axs\.tv/(?:channel/[^/?#]+/)?video/(?P<id>[^/?#]+)'
```

Reply:
Good catch. Missing the `#` was bad inspiration from other extractors, missing it in the optional path insert was an oversight.

I also needed to adapt the URL regex for one further depth of path (`channel/series/video/`).